app/assets/stylesheets/snowcrash/modules/notifications.scss

```diff
@@ -37,7 +36,7 @@
       li.notification {
         position: relative;
```

Comment:
It might not be directly relevant to your comment, but don't forget that the "notification" CSS class is also used (or at least will be used) by the `notifications#index` page. (See #315)